lib/services/apptweak_api/metadata.rb

```diff
@@ -0,0 +1,6 @@
+class Services < Services::ApptweakApi::Base
```

Comment:
а это точно работает?
может вот так правильно? 
```suggestion
 module Services
  module ApptweakApi
    class Metadata < Services::ApptweakApi::Base
```

Reply:
ИСправь везде